setup.py

```diff
@@ -73,6 +73,7 @@
         "lmdb",
         "more-itertools",
         "wandb>=0.10.0,<0.11.0",
+        "datasets==1.5.0",
```

Comment:
Do we have to pin the precise version? Can we be at least a little flexible? We don't know what else people's projects will depend on, so giving a little flexibility helps people out.

Reply:
Cool. Something we do sometimes, like in the `wandb` case above, is pin it to a `major.minor` version, but leave the patch version open. In this case that would be `>=1.5.0,<1.6.0`. Then at least we automatically get little fixes that huggingface provides.

Reply:
I have updated this to be the same, hope not testing it with older versions is okay for now.